### DISCUSSION TITLE
Repository status

### DISCUSSION THREAD

**Discussion by ttytm** [akiyosi/goneovim#447] [Q&A] [answered]
If I recall it right, the repository was a fork some time ago. Back then, I thought it was a pity, as it had clearly become its own project over time. Now it is.

I didn't think that was possible, even when I spoke with a GitHub internal that's what I was told. But it's been a while since then and things may have changed. 

For a fellow dev who's simply curious, could you shed some light on this if the repo was converted?

**Comment by akiyosi** [comment, marked as answer]
@tobealive 
Yes. As you understand, this project was renamed in the first phase and changed to an independent project in the second phase by removing the fork label, as it has undergone significant changes from the original gonvim.
The removal of the fork label can be done officially on Github, although it requires assistance from the engineers inside Github. See below.

https://support.github.com/request/fork
